lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -80,6 +80,24 @@ bool stringBeginsWithDigit(const std::string& inputString)
     return false;
 }
 
+// find position index for a given layer in the local LayerStack ( strong-to-weak order )
+uint32_t findLayerIndex(const PXR_NS::UsdAttribute& attr, const PXR_NS::SdfLayerHandle& layer)
```

Comment:
Just noticed this (sorry...), and not a big deal because it's in the unnamed namespace, but interface is awkward: you require an attr, but in fact you really want just a stage, because you use the attr only to get the stage.  If we ever factor this out, I would change this to accept a stage as the first argument.

Reply:
addressed ac2bb4dc732e9862957b25a17eb9f6f7719adcf3